tests/components/deconz/test_binary_sensor.py

```diff
@@ -67,6 +67,7 @@ async def test_no_binary_sensors(hass):
     """Test that no sensors in deconz results in no sensor entities."""
     gateway = await setup_deconz_integration(hass)
     assert len(gateway.deconz_ids) == 0
+    assert len(gateway.entities[binary_sensor.DOMAIN]) == 0
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6239266)